stories/feed-list/feed-list.stories.js

```diff
@@ -0,0 +1,562 @@
+                    </div>
+                    <div class="fd-popover__body fd-popover__body--right" aria-hidden="true" id="actionSheetDesktop">
+                        <ul class="fd-action-sheet fd-action-sheet--compact" role="list" aria-label="List of contextual options">
+                            <li class="fd-action-sheet__item" role="listitem">
+                                <button class="fd-button fd-button--full-width fd-button--compact fd-button--transparent fd-button--text-alignment-left">
+                                    <i class="sap-icon--attachment"></i>
+                                    <span class="fd-button__text">Button 1</span>
+                                </button>
+                            </li>
+                            <li class="fd-action-sheet__item" role="listitem">
+                                <button class="fd-button fd-button--full-width fd-button--compact fd-button--transparent fd-button--text-alignment-left">
+                                    <i class="sap-icon--add"></i>
+                                    <span class="fd-button__text">Button 2</span>
+                                </button>
+                            </li>
+                            <li class="fd-action-sheet__item" role="listitem">
+                                <button class="fd-button fd-button--full-width fd-button--compact fd-button--transparent fd-button--text-alignment-left">
+                                    <i class="sap-icon--print"></i>
+                                    <span class="fd-button__text">Button 3</span>
+                                </button>
+                            </li>
+                        </ul>
+                    </div>
+                </div>
+            </div>
+            <p class="fd-feed-list__text">
+                <span class="fd-feed-list__name">Joe Doe: </span>
+                <span>In hac habitasse platea dictumst. Fusce eu dui viverra, dictum justo in, pul aug. Praesent aliquam massa non lectus commodo, id vestibulum lectus auctor. Proin lorem quam, accumsan ac dui et, cursus ornare dui. Nunc ultricies dolor felis, in viverra mi venenatis in. Sed vel nisl a dui posuere ullamcorper. Donec posuere id massa at dictum. Suspendisse potenti.</span>
+            </p>                 
+            <div class="fd-feed-list__footer">
+                <small class="fd-feed-list__footer--byline">Aug 20, 2020</small>
+                <small class="fd-feed-list__footer--byline">Reply</small>
+            </div>
+        </div>        
+    </li>
+</ul>
+`;
+
+withActions.storyName = 'Actions';
+withActions.parameters = {
+    docs: {
+        storyDescription: 'Feed list items can display actions that users can perform on their individual feed posts. These typically include actions like Edit and Delete, however, other actions can be displayed depending on the use case. It is recommended to display a maximum of 5 actions per post.'

```

Comment:
Italicize "Edit" and "Delete" like this: 

These typically include actions like _Edit_ and _Delete_